msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md

```diff
@@ -172,6 +172,7 @@ The configuration page code informs Teams that the configuration requirements ar
 >* You have 30 seconds to complete the save operation (the callback to registerOnSaveHandler) before the timeout. After the timeout, a generic error message appears.
```

Comment:
While we are at it, let's use backticks for `registerOnSaveHandler` and other such params.